pkg/util/mount/linux.go

```diff
@@ -47,10 +50,44 @@ func (mounter *mounter) Unmount(target string, flags int) error {
 	return syscall.Unmount(target, flags)
 }
 
+// safe_format_and_mount is a utility script on GCE VMs that probes a persistent disk, and if
+// necessary formats it before mounting it.
+// This eliminates the necesisty to format a PD before it is used with a Pod on GCE.
+// TODO: port this script into Go and use it for all Linux platforms
+type gceSafeFormatAndMount struct {
```

Comment:
I started this prior to your plugin refactor, so I haven't linked it into the code yet.  That's the next PR.  I can do it in this PR if you prefer.